client/src/components/molecules/AccountStatus/Popover.module.scss

```diff
@@ -7,12 +7,12 @@ $popoverWidth: 18rem;
     width: $popoverWidth;
     padding: $spacer / 2;
     background: $brand-black;
-    border-radius: .1rem;
-    border: .1rem solid $brand-grey-light;
-    box-shadow: 0 6px 16px 0 rgba($brand-black, .3);
+    border-radius: 0.1rem;
+    border: 0.1rem solid $brand-grey-light;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)